examples/runDecamQuickTest.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Comments below apply to all the `runXXXTest.sh` scripts.

There's still a lot of duplication between them, although (as per the following) I think you could actually remove much of the code so maybe that's not a big deal. They seem to be crying out for command line arguments, though (wouldn't `runDecamTest.sh --quick` be nicer than a separate script?!).